repose-aggregator/external/pjl-compressing-filter/src/test/java/org/openrepose/external/pjlcompression/LogTest.java

```diff
@@ -49,21 +48,6 @@
  */
 public final class LogTest {
 
-    /**
-     * this test tests a commons logging implementation that we are *NOT* using
-     *
-     * @throws Exception
-     */
-    @Test
-    @Ignore
-    public void testJakartaImpl() throws Exception {
```

Comment:
Pretty sure this isn't necessary. We don't use the Jakarta logging for anything, and a search in IntelliJ doesn't show that class as existing on the classpath. Also, reflectively testing a constructor? No idea what the goal was there.

If I'm wrong, though, someone can correct me.